tests/golden/docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  collector:
+    container_name: collector
+    image: ${IMG}
+    volumes:
+      - ./config.yaml:/etc/config.yaml
+    command:
+      - --config
+      - /etc/config.yaml
+  golden:
+    container_name: golden
+    image: ghcr.io/open-telemetry/opentelemetry-collector-contrib/golden:latest
```

Comment:
where is this image coming from? 🤔 
is it update automatically somewhere when new contrib versions are released?

Reply:
ah, found it already: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/workflows/golden.yml